### PR TITLE
added makere

### PR DESCRIPTION
**Pull request by aksJain0**
makere generates a regex satisfying a list of strings that is given to it.

### CODE REVIEW

File: makere/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+## Design Thoughts
+As the problem statement states that strings with similar pattrens should pass
+but the similarities may be of different kind and may also sometimes depend on the context to which the strings are from.
+
+So we will be grouping letters to basic regex classes which are as follows:
+* [a-z]
+* [0-9]
+* [A-Z]
+
+But this way we won't be able to fing common substrings, to resolve that issue,
```
Comment (adityaarakeri) [inline makere/README.md]: typo `fing`

File: makere/setup.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from setuptools import setup
+
+setup ( name = 'makere',
+        version = '0.1',
+        description='Library to convert list of strings to regex',
+        author = 'Akshay Jain',
+        author_email='aakshay1294@gmail.com',
+        packages = ['makere'],
+         install_requires=[        
+      ],)
```
Comment (adityaarakeri) [inline makere/setup.py]: Can you make the setup specific to python3? since python2.7 will be EOL by this year end

**Comment by adityaarakeri** [comment]
also update the `root` Readme file to add your project to the list of projects

File: makere/README.md (hunk 1 above)
Comment (aksJain0) [reply makere/README.md]: fixed

File: makere/setup.py (hunk 2 above)
Comment (aksJain0) [reply makere/setup.py]: Supports both python2 and 3 now

**Comment by aksJain0** [comment]
Please review the changes

**Pull request merged**